docs/user/reporting/index.asciidoc

```diff
@@ -27,6 +27,10 @@ You access the options from the *Share* menu in the toolbar. The sharing options
 
 * *Embed code* &mdash; Embed a fully interactive dashboard or visualization as an iframe on a web page.
 
+[[reporting-on-cloud-resource-requirements]]
+NOTE: On Elastic Cloud *PDF* and *PNG* reports require a Kibana with minimum 2GB RAM. Visual reports
```

Comment:
Can we make it one sentence?

> On Elastic Cloud, Kibana requires a minimum of 2GB RAM to generate quality PDF and PNG reports.

Reply:
The missing piece with that iteration is that reports require 2GB RAM to work, not just that they'll be of better quality. Perhaps we can drop the point about stability as that is entailed by "minimum".

Reading this again, I think we should call out the fact that users can do this from the Cloud UI. How about:

"
On Elastic Cloud, Kibana requires a minimum of 2GB RAM to generate PDF or PNG reports. Kibana deployments can be edited from the [Cloud UI](link-somehwere).
"

Still at 2 sentences 😅 . Let me know what you think!

(also do we ever link to Cloud Management UI from the docs?)

Reply:
Yes, I like your suggestion.  We link to Elastic Cloud from the [Kibana Quickstart](https://www.elastic.co/guide/en/kibana/current/get-started.html).

Is Cloud Management the correct name of the UI?

> On Elastic Cloud, Kibana requires a minimum of 2GB RAM to generate PDF or PNG reports. To edit Kibana deployments, use [Cloud Management](https://github.com/elastic/kibana/pull/link-somehwere).

where Cloud Management links to the appropriate doc.

Reply:
This looks great! Thanks for the reference!